index.html

```diff
@@ -64,14 +63,168 @@ <h3>
               <a href="#"><i class="fa-brands fa-angellist"></i></a>
             </li>
           </ul>
-        </div>
+        </section>
         <img src="images/Action.svg" alt="action" />
         <div class="my-recent-works">
-          <h3>My Recent</h3>
-          <h3>Works</h3>
-          <img src="images/Indicator.png" alt="Indicator">
+          <h3>My Recent Works</h3>
+          <img src="images/Indicator.svg" alt="Indicator" />
```

Comment:
- [ ] (Optional) It would be nice for you to make the color of the `<h3>`  `#172b4d` instead of black.